packages/shared/src/components/PostOptionsMenu.tsx

```diff
@@ -39,7 +39,8 @@ export default function PostOptionsMenu({
   onMessage,
   onRemovePost,
 }: PostOptionsMenuProps): ReactElement {
-  const { refreshFeed } = useFeedSettings();
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const { feedSettings } = useFeedSettings();
```

Comment:
If you want to fix the lint issue, this would do

```suggestion
useFeedSettings();
```